msal/authority.py

```diff
@@ -69,9 +69,16 @@ def __init__(self, authority_url, http_client, validate_authority=True):
                     authority.path,  # In B2C scenario, it is "/tenant/policy"
                     "" if tenant == "adfs" else "/v2.0" # the AAD v2 endpoint
                     ))
-        openid_config = tenant_discovery(
-            tenant_discovery_endpoint,
-            self.http_client)
+        try:
```

Comment:
Any specific reason why we catch it here and not where it originates , in line 132 ? Just curious as I went searching for  where this originates and considered the possibility of having this same try-except block there

Reply:
It is mainly because the input `authority_url` would not be available there.